Tests/Integration/BehaviorTests/run_behavior_tests.sh

```diff
@@ -3,7 +3,7 @@
 NUM_FAILED=0
 
 echo "Compiling Flint sources"
-./compile_behavior_tests.sh
+#./compile_behavior_tests.sh
```

Comment:
This is not how you pass CI.
Can you please look at the diff before making a commit? :|